tests/core/readers/conftest.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from datetime import date, timedelta
+
+from pyramid_oereb.core import b64
+from pyramid_oereb.core.adapter import FileAdapter
+from pyramid_oereb.contrib.data_sources.standard.sources.plr import StandardThemeConfigParser
+
+
+file_adapter = FileAdapter()
+
+
+@pytest.fixture
+def wms_url_contaminated_sites():
+    return {
+        'de': 'https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&'
+              'CRS=EPSG:2056&BBOX=2475000,1065000,2850000,1300000&WIDTH=740&HEIGHT=500&FORMAT=image/png'
+              '&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb',
+        'fr': 'https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&'
+              'CRS=EPSG:2056&BBOX=2475000,1065000,2850000,1300000&WIDTH=740&HEIGHT=500&FORMAT=image/png'
+              '&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb'
+            }
+
+
+@pytest.fixture
+def land_use_plans(pyramid_oereb_test_config, dbsession, transact, wms_url_contaminated_sites):
+    del transact
+
+    theme_config = pyramid_oereb_test_config.get_theme_config_by_code('ch.Nutzungsplanung')
+    config_parser = StandardThemeConfigParser(**theme_config)
+    models = config_parser.get_models()
+
+    view_services = {
+        'view_service99': models.ViewService(
+            id=99,
+            reference_wms=wms_url_contaminated_sites,
+            layer_index=1,
+            layer_opacity=.75,
+        )
+        }
+    dbsession.add_all(view_services.values())
```

Comment:
for all occurrences of add_all:
is there a benefit of creating a dict and then using .values() to get the value iterator ?

Wouldn't it be easier to use a list like:
```
   view_services = [
        models.ViewService(
            id=99,
            reference_wms=wms_url_contaminated_sites,
            layer_index=1,
            layer_opacity=.75,
        )
        }
    dbsession.add_all(view_services)
```

Reply:
or even organize the imports differently so that it is possible to copy paste the code and only make a search - replace operation:

my idea:
original: 
```
        connection.execute(contaminated_sites.Office.__table__.insert(), {
            'id': '1',
            'name': {'de': u'Test Office'}
        })
```
fixture:
```
        dbsession.add(Office(**{
            'id': '1',
            'name': {'de': u'Test Office'}
        })
```
so you only have to replace `.__table__.insert(), ` by `(**` and `connection.execute` by `dbsession.add(`
and somehow mange the model import...

Reply:
I think that joins the idea from Clemens as I understood it: having always the same kind of load process and to put the json part to fill the db tables in a file. But I might be wrong with my understanding @vvmruder

Reply:
That would indeed save a lot of time for the migration and it seems it would still be possible to use the fixture for asserts in the test - new PR https://github.com/openoereb/pyramid_oereb/pull/1429/

Reply:
Yes. The idea was to have data (also for tests) in the database already. This way we can improve it easily and even see it through e.g. QGIS. Should I support you by adding the data as JSONS into the sampte-data structure?

Reply:
This morning in the meeting the consensus was a different one... But we can rediscuss it if needed, a live meeting would be needed I think.

Reply:
Ah-ok then nevermind

Reply:
I only thought we want get rid of the in python dynamic db interactions.

Reply:
@vvmruder we started heading into the direction where CI tests are conducted on a lightweight disposable DB. See PR https://github.com/openoereb/pyramid_oereb/pull/1431 for a fixture of this temp DB.
This temporary DB can also be consulted from QGIS if:
- the transaction - rollback mechanism in the tests is deactivated so that everything is committed to the DB (could be a parameter when launching the test script)
- a breakpoint is set somewhere in the test sequence before the DB is dropped. (or we could introduce a parameter like DB_NO_DROP)
I consider this type of tests suitable for CI since setup and teardown are really fast and few data needs to be put into the repo.

However it does not exclude functional tests on a larger, static DB created from json files. But maybe these tests should not be run systematically in CI ?

It would be good to discuss and clarify this point quite quickly, hopefully we can organize a meeting with Wolfgang.

Reply:
No its ok for me. I agree on this approach if ti fits the needs :)